### PR TITLE
Add QuickSortRandomPivot.java

### PR DESCRIPTION
**Pull request by the-cybersapien**
Added java implementation for QuickSort using Random Pivot

### CODE REVIEW

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java]: No need for this new line.

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Quick sort algorithm with random pivot selection
+http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/
+GitHub - https://github.com/VamsiSangam/theoryofprogramming
+Code Contributor - Aditya Aggarwal (github.com/the-cybersapien, cybersapien.xyz)
+
+===== ===== ===== */
+
+import java.util.Scanner;
+
+
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java]: No need for this new line.

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+public class Solution {
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java]: Please name the class ```QuickSortRandomPivot```.

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+        for (int a : array) {
+            System.out.print(a + " ");
+        }
+
+        System.out.println();
+    }
+
+    private static void quickSortRandomPivot(int[] array, int low, int high){
+
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java]: No need for this new line.

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java (hunk 3 above)
Comment (the-cybersapien) [reply Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java]: Sorry, errors like these happen when one uses an IDE to code and then use a text editor to add text and create a file :laughing: 
Fixed it. Please review.

**Review by VamsiSangam — APPROVED** [review]

**Comment by VamsiSangam** [comment]
Thanks! Merging the PR. You should be able to see your code live on the [quick sort](http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/) post in a few minutes. 😄

**Pull request merged**